docs/sources/image-rendering/_index.md

```diff
@@ -0,0 +1,401 @@
+
+Default mode will create a new browser instance on each request. When handling multiple concurrent requests, this mode increases memory usage as it will launch multiple browsers at the same time. If you want to set a maximum number of browser to open, you'll need to use the [clustered mode](#clustered).
+
+> **Note:** When using the `default` mode, it's recommended to not remove the default Chromium flag `--disable-gpu`. When receiving a lot of concurrent requests, not using this flag can cause Puppeteer `newPage` function to freeze, causing request timeouts and leaving browsers open.
+
+```bash
+RENDERING_MODE=default
+```
+
+```json
+{
+  "rendering": {
+    "mode": "default"
+  }
+}
+```
+
+#### Clustered
+
+With the `clustered` mode, you can configure how many browser instances or incognito pages can execute concurrently. Default is `browser` and will ensure a maximum amount of browser instances can execute concurrently. Mode `context` will ensure a maximum amount of incognito pages can execute concurrently. You can also configure the maximum concurrency allowed, which per default is `5`.
+
+Using a cluster of incognito pages is more performant and consumes less CPU and memory than a cluster of browsers. However, if one page crashes it can bring down the entire browser with it (making all the rendering requests happening at the same time fail). Also, each page isn't guaranteed to be totally clean (cookies and storage might bleed-through as seen [here](https://bugs.chromium.org/p/chromium/issues/detail?id=754576)).
+
+```bash
+RENDERING_MODE=clustered
+RENDERING_CLUSTERING_MODE=browser
+RENDERING_CLUSTERING_MAX_CONCURRENCY=5
+```
+
+```json
+{
+  "rendering": {
+    "mode": "clustered",
+    "clustering": {
+      "mode": "browser",
+      "maxConcurrency": 5
+    }
+  }
+}
+```
+
+#### Reusable (experimental)
+
+When using the rendering mode `reusable`, one browser instance will be created and reused. A new incognito page will be opened for each request. This mode is experimental since, if the browser instance crashes, it will not automatically be restarted. You can achieve a similar behavior using `clustered` mode with a high `maxConcurrency` setting.
+
+```bash
+RENDERING_MODE=reusable
+```
+
+```json
+{
+  "rendering": {
+    "mode": "reusable"
+  }
+}
+```
+
+#### Rendering modes performance and CPU / memory usage
+
+The performance and resources consumption of the different modes depend a lot on the number of concurrent requests your service is handling so it's recommended to [monitor your image renderer service]({{< relref "./monitoring/" >}}).
+
+With no concurrent request, the different modes show very similar performance and CPU / memory usage.
+
+When handling concurrent requests, we see the following trends:
+
+- Parallelizing incognito pages instead of browsers is better for performance and CPU / memory consumption meaning that [clustered](#clustered) (with clustering mode set as `context`) mode is the most performant.
```

Comment:
```suggestion
- To improve performance and reduce CPU and memory consumption, use [clustered](#clustered) mode with `RENDERING_CLUSTERING_MODE` set as `context`. This parallelizes incognito pages instead of browsers.
```